runtime/compiler/runtime/IProfiler.cpp

```diff
@@ -1208,6 +1208,10 @@ TR_IPMethodHashTableEntry::add(TR_OpaqueMethodBlock *caller, TR_OpaqueMethodBloc
 bool
 TR_IProfiler::invalidateEntryIfInconsistent(TR_IPBytecodeHashTableEntry *entry)
    {
+   // Invalid entries are purged elsewhere in JITaaS server mode
+   if (_compInfo->getPersistentInfo()->getJITaaSMode() == SERVER_MODE)
```

Comment:
Instead of this check we could make this method virtual and redefine it as an empty one in TR_JITaaSIProfiler